bot/processors/matchmaking.py

```diff
@@ -64,6 +65,43 @@ def confirm_match_request(channel_id):
     return blocks
 
 
+def create_match_blocks(topic_channel_id, conversation_id):
+    blocks = [
+        {
+            "type": "section",
+            "text": {
+                "type": "mrkdwn",
+                "text": f"Yahoo, you've been matched for a conversation about <#{topic_channel_id}>!",
```

Comment:
What they click on originally to opt-in actually uses the channel name:
```
{
    "type": "section",
    "text": {
        "type": "mrkdwn",
        "text": f"Hey folks! <#{channel_id}> has been set up as a topic channel. Every couple of weeks, "
                f"I (the Penny Bot) will set you up with other users to chat about <#{channel_id}>.",
    }
},
```
While I like the idea of being able to pass any string there, I think we should stick to it being the topic channel for the sake of simplicity. All of the Match objects have a foreign key to a topic channel, so it limits it to that. 

I'm a little bit confused by some of your comments, but maybe we are thinking about TopicChannels differently. To me, the channel itself _is_ a topic, that's why they are called TopicChannels. The name of the channel indicates what the topic is. So it makes sense to use the channel name when matching the person. I'm not quite sure what the difference is between `"Yahoo, you've been matched for a conversation about <#{topic}>!"` and `"Yahoo, you've been matched for a <#{topic_channel_id}> discussion."`. I'm not partial to either verbiage, but it sounds like you thought the original was confusing and I'm not getting how the latter clears anything up (or what exactly you're confused about).

Reply:
> To me, the channel itself is a topic, that's why they are called TopicChannels. The name of the channel indicates what the topic is.

Perfect. Let's go with that. And do whatever you wish with the "Yahoo" verbiage.

The confusion I was having has to do with the `/penny set-topic` command because I thought you used to have to actually set the topic... but looking at the code I'm apparently fabricating that memory.

All good!